tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -76,6 +76,10 @@ et build //flutter/fml:fml_benchmarks  # Build a specific target in `//flutter/f
       if (useLto) '--lto' else '--no-lto',
     ];
 
+    if (!await ensureBuildDir(environment, build, enableRbe: useRbe)) {
```

Comment:
This is only needed if `argResults.rest` is non-empty since the first thing that `runBuild` is going to do is run `gn`.

Reply:
Good catch, done!